pages/mi/mi-github-issues.md

```diff
@@ -47,18 +47,25 @@ Before you create your branch, sync your repo using the commands:
 `git push origin master`
 More details in the [previous step](mi-github-and-repositories.md#Summary_of_Steps).
 
-To create a new branch, go to your github repo and create it in a similar fashion as the following image:
```

Comment:
From the content perspective, as we discussed in the original issue, I removed line 50-51 since they are causing confusion.